community-operators/dynatrace/dynatrace-monitoring.v0.2.0.clusterseviceversion.yaml

```diff
@@ -0,0 +1,215 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      {
+      	"apiVersion": "dynatrace.com/v1alpha1",
+      	"kind": "OneAgent",
+      	"metadata": {
+      		"name": "oneagent",
+      		"namespace": "dynatrace"
+      	},
+      	"spec": {
+      		"apiUrl": "https://ENVIRONMENTID.live.dynatrace.com/api",
+      		"skipCertCheck": false,
+      		"tokens": "",
+      		"nodeSelector": {},
+      		"tolerations": [
+      			{
+      				"effect": "NoSchedule",
+      				"key": "node-role.kubernetes.io/master",
+      				"operator": "Exists"
+      			}
+      		],
+      		"image": "",
+      		"args": [
+      			"APP_LOG_CONTENT_ACCESS=1"
+      		],
+      		"env": []
+      	}
+      }
+    categories: "Monitoring"
+    certified: "false"
+    containerImage: quay.io/dynatrace/dynatrace-oneagent-operator:v0.2.0
+    createdAt: 2019-02-06T12:59:59Z
+    description: Install full-stack monitoring of OpenShift clusters with the Dynatrace OneAgent.
+    support: Dynatrace
+  generation: 1
```

Comment:
What does this tag do?

Reply:
> The `.metadata.generation` value is incremented for all changes, except for changes to .metadata or .status.

Do I need to keep this value or should I just remove it?

Reply:
I don't know, please look into it.

Reply:
I think this should be removed, it's added by the cluster automatically.